lib/network_ssl/network_ssl_compat.c

```diff
@@ -63,6 +64,54 @@ network_ssl_compat_CTL_set_min_proto_version(SSL_CTX * ctx, int version)
 }
 #endif
 
+#ifdef NETWORK_SSL_COMPAT_CHECK_HOSTNAME
+/**
+ * network_ssl_compat_set1_host(ssl, hostname):
+ * Set expected name for hostname verification.
+ *
+ * COMPATIBILITY: provides SSL_set1_host().
+ */
+int
+network_ssl_compat_set1_host(SSL * ssl, const char * hostname)
+{
+
```

Comment:
Why the whitespace?

Reply:
left over from the `#if` days.  Removed.